addons/portal/__manifest__.py

```diff
@@ -39,18 +38,74 @@
         'web.assets_frontend': [
             'portal/static/src/scss/portal.scss',
             'portal/static/src/js/portal.js',
-            'portal/static/src/js/portal_chatter.js',
             'portal/static/src/xml/portal_chatter.xml',
             'portal/static/src/js/portal_composer.js',
             'portal/static/src/js/portal_security.js',
             'portal/static/src/js/portal_sidebar.js',
             'portal/static/src/xml/portal_security.xml',
             'portal/static/src/js/components/**/*',
             'portal/static/src/signature_form/**/*',
+            'portal/static/src/chatter/boot/boot_service.js',
         ],
         'web.assets_tests': [
             'portal/static/tests/**/*',
         ],
+        "portal.assets_chatter_helpers": [
+            "web/static/src/views/view_dialogs/form_view_dialog.js",
+            "web/static/src/core/debug/*",
+            "web/static/src/core/commands/command_hook.js",
+            "web/static/src/model/**/*",
+            "web/static/src/search/**/*",
+            "web/static/src/views/view.js",
+            "web/static/src/views/view_hook.js",
+            "web/static/src/webclient/actions/action_dialog.js",
+            "web/static/src/webclient/actions/reports/utils.js",
+            "web/static/src/webclient/actions/reports/report_action.js",
+            "web/static/src/webclient/actions/reports/report_hook.js",
+            "web/static/src/views/utils.js",
+            "web/static/src/views/fields/formatters.js",
+            "web/static/src/views/fields/file_handler.*",
+            "mail/static/src/model/**/*",
+            "mail/static/src/core/common/**/*",
+            "mail/static/src/core/web_portal/**/*",
+            "mail/static/src/utils/common/**/*",
+            "mail/static/src/chatter/web_portal/**/*",
+            "mail/static/src/discuss/typing/**/*",
+            "mail/static/src/discuss/core/common/action_panel.js",
+            ("remove", "mail/static/src/**/*.dark.scss"),
+        ],
+        "portal.assets_chatter": [
+            ("include", "web._assets_helpers"),
+            ("include", "web._assets_frontend_helpers"),
+            "web/static/src/scss/pre_variables.scss",
+            "web/static/lib/bootstrap/scss/_variables.scss",
+            "web/static/lib/bootstrap/scss/_variables-dark.scss",
+            ("include", "portal.assets_chatter_helpers"),
+            "portal/static/src/chatter/core/**/*",
+            "portal/static/src/chatter/frontend/**/*",
+        ],
+        "portal.assets_embed": [
```

Comment:
I would rename this to something else, as it is confusing with what we do in livechat.
`assets_chatter_style` maybe?

Shouldn't scss files excluded from other chatter portal bundles then?